src/Hyperwallet/Response/ListResponse.php

```diff
@@ -6,14 +6,28 @@
  *
  * @package Hyperwallet\Response
  */
-class ListResponse implements \Countable, \ArrayAccess {
+class ListResponse implements \Countable , \ArrayAccess{
 
     /**
      * Total number of matching objects
      *
      * @var int
      */
-    private $count;
+    private $limit;
+
+    /**
+     * Has Next Page objects
+     *
+     * @var boolean
+     */
+    private $hasNextPage;
+
+    /**
+     * Has Next Page objects
+     *
+     * @var boolean
```

Comment:
Pls update this comment - has Previous page

Reply:
This changes has been done , please check below commit id - 2c27e80a631550d33f0c3bddc01ecb067c5b008b